src/pages/eunjeonglee/Detail/Detail.js

```diff
@@ -1,8 +1,196 @@
 import React from 'react';
+import TopNav from '../TopNav/TopNav';
+import HandleLike from '../HandleLike/HandleLike';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart as regularHeart } from '@fortawesome/free-regular-svg-icons';
+import {
+  faHeart as solidHeart,
+  faTimes,
+} from '@fortawesome/free-solid-svg-icons';
+import './Detail.scss';
 
 class Detail extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      likeHeart: false,
+      reviewValue: '',
+      commentList: [],
+    };
+  }
+
+  handleLike = event => {
+    const { likeHeart } = this.state;
+    this.setState({
+      likeHeart: !likeHeart,
+    });
+  };
+
+  getValue = event => {
+    const { value } = event.target;
+    this.setState({
+      reviewValue: value,
+    });
+  };
+
+  pushReview = event => {
+    const { commentList } = this.state;
+    if (event.key === 'Enter') {
+      commentList.push(this.state.reviewValue);
+      event.target.value = '';
+      this.setState({
+        reviewValue: '',
+      });
+    }
+  };
+
+  deleteReview = idx => {
+    const { commentList } = this.state;
+    commentList.splice(idx, 1);
+    this.setState({
+      commentList: commentList,
```

Comment:
두 개가 같다면 하나로 생략도 가능!
```suggestion
      commentList,
```